libmdnsd/CMakeLists.txt

```
@@ -63,13 +67,33 @@ if(MDNSD_COMPILE_AS_CXX)
 endif()
 
 if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES})
+    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES})
+    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 endif()
 
 if(WIN32 OR MINGW)
```

`WIN32` is also true for mingw.
```suggestion
if(WIN32)
```